scripts/generator/generator/pyclblast.py

```diff
@@ -0,0 +1,113 @@
+
+# This file is part of the CLBlast project. The project is licensed under Apache Version 2.0. This file follows the
+# PEP8 Python style guide and uses a max-width of 120 characters per line.
+#
+# Author(s):
+#   Cedric Nugteren <www.cedricnugteren.nl>
+
+NL = "\n"
```

Comment:
What about using os.linesep? A little wordy but you can alias it to NL. I suppose this only runs on linux but might be useful in future.